conference/react-js/src/containers/Talks/FilterForm/index.jsx

```diff
@@ -1,52 +1,48 @@
 import React, { PureComponent } from 'react';
 import { PropTypes } from 'prop-types';
-import ReactInputSelect from 'react-input-select/lib/ReactInputSelect';
 
 class FilterForm extends PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      topicValue: '',
-      listOfTopics: [],
-    };
-  }
-
-  componentWillReceiveProps(nextProps) {
-    const { topics } = nextProps;
-    const listOfTopics = topics.map(({ name }) => name);
-    this.setState({ listOfTopics });
-  }
-
-  onBlurTopic = (e) => {
-    const name = 'topic';
-    const value = e.target.value;
-    this.props.onChangeFilter({ target: { name, value } });
-  }
-
-  updateTopicValue = (e) => {
-    this.setState({ topicValue: e.target.value });
-  }
-
-  chooseTopic = (item) => {
-    this.setState({ topicValue: item });
-  }
-
-  resetTopics = () => {
-    this.props.handleResetFiltersClick();
-    this.setState({ topicValue: '' });
-  }
+  setStatus = status => (status.map(element =>
```

Comment:
as I've noticed, this method returns a list of elements(options) so the name `getOptions` will mach better